NewHorizons/Utility/Files/ImageUtilities.cs

```diff
@@ -136,13 +132,20 @@ public static Texture2D InvertSlideReel(IModBehaviour mod, Texture2D texture, st
                 }
                 else
                 {
-                    pixels[i].r = 1f - pixels[i].r;
-                    pixels[i].g = 1f - pixels[i].g;
-                    pixels[i].b = 1f - pixels[i].b;
+                    // convert gamma to linear, then invert
+                    // outer wilds will invert, then convert linear to gamma (reversing the process)
+                    pixels[i].r = 1f - Mathf.GammaToLinearSpace(pixels[i].r);
+                    pixels[i].g = 1f - Mathf.GammaToLinearSpace(pixels[i].g);
+                    pixels[i].b = 1f - Mathf.GammaToLinearSpace(pixels[i].b);
                 }
             }
 
-            var newTexture = new Texture2D(texture.width, texture.height, texture.format, texture.mipmapCount != 1, true);
+            // normally, you would want to keep this linear because we do our own gamma to linear conversion.
+            // (it also makes vanilla reel atlas match vanilla reels)
+            // however, not doing linear makes it match more closely to the source image, which is more desireable
+            // change linear to true if someone complains
```

Comment:
why would they complain? altho the comparison looks closer when linear is set to true so I dont understand why we dont just do that?

Reply:
when linear is true it works for base game atlas images, but it makes it _darker than the source image_, which on custom slide reels can looks silly

Reply:
ill update the comment to mention that. the comment i posted above shows that matching the source image is nicer. i also tested it with other images and the colors look off when linear is true (its too dark)